conf/config.json

```diff
@@ -37,6 +37,11 @@
             "readPreference": "primary",
             "database": "metadata"
         },
+        "kafka": {
```

Comment:
this is not (IMHO) a separate kafka host : we should use the one configured "globally" for backbeat.
The only thing we need here is indeed the `topic` & `groupId` (which may be renamed `consumerGroupId`, for clarity ?)